MAPL_Base/MAPL_EtaHybridVerticalCoordinate.F90

```diff
@@ -1,6 +1,6 @@
 #include "MAPL_Generic.h"
 
-module MAPL_EtaHybridVerticalCoordinate
+module MAPL_EtaHybridVerticalCoordinateMod

```

Comment:
Note that the _Mod suffix will be going away in newer versions of MAPL.   But not worth fixing back now.